data_managers/data_manager_build_kraken_database/tool_dependencies.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<tool_dependency>
+    <set_environment version="1.0">
+        <environment_variable action="set_to" name="KRAKEN_SCRIPT_PATH">$REPOSITORY_INSTALL_DIR</environment_variable>
```

Comment:
``_SCRIPT_PATH`` is a deprecated idiom - I would eliminate this and just use ``$__tool_directory__`` in the tool.

For more information - see https://bitbucket.org/galaxy/galaxy-central/commits/ddbe0d12bc86da406bb96213e23b8ddb273ed669 and bgruening/galaxytools#85.